frontend/src/routes/_oh/route.tsx

```diff
@@ -47,7 +47,7 @@ export default function MainApp() {
   const { data: settings } = useSettings();
 
   const [consentFormIsOpen, setConsentFormIsOpen] = React.useState(
-    !localStorage.getItem("analytics-consent"),
+    !localStorage.getItem("consent-form-closed"),
```

Comment:
this seems like everyone will see the consent modal on first load now? Even people who already made a choice?